Packs/CloudShare/Integrations/CloudShare/CloudShare.yml

```
@@ -0,0 +1,2683 @@
+      description: Accept invitation URL
+      type: string
+    - contextPath: CloudShare.Invites.message
+      description: Message
+      type: string
+    description: Invite an end user to a POC, based on a specified blueprint, and
+      assigns a specific project member user to be the owning project member for the
+      end user.  A policy ID, blueprint ID and owning project member ID must be specified.
+  - name: cloudshare-get-poc-invitations
+    arguments:
+    - name: from
+      description: 'Retrieve invites from a particular time (Example: "last 7 days")'
+    - name: sentBy
+      description: Filter based on who sent the invite (default is anyone).
+    - name: skip
+      description: 'Specifies to skip the first {skip} records, where {skip} is an
+        integer (default: 0). Can be used iteratively in conjunction with take to
+        view distinct sets of template records.'
+    - name: sortBy
+      description: The filed to sort by
+    - name: statusCategory
+      description: The status categoey to filter by
+    - name: take
+      description: 'Limits the number of records returned, where {take} is the maximum
+        number of records to return. Integer (default: 1000, maximum: 1000).'
+    outputs:
+    - contextPath: CloudShare.POCInvites.blueprintShortId
+      description: Blueprint short ID
+      type: string
+    - contextPath: CloudShare.POCInvites.projectName
+      description: Project name
+      type: string
+    - contextPath: CloudShare.POCInvites.createdOn
+      description: Created on
+      type: string
+    - contextPath: CloudShare.POCInvites.owningSe
+      description: Owning SE
+      type: string
+    - contextPath: CloudShare.POCInvites.environmentId
+      description: Environment ID
+      type: string
+    - contextPath: CloudShare.POCInvites.pocUser
+      description: POC user
+      type: string
+    - contextPath: CloudShare.POCInvites.startDate
+      description: Start date
+      type: string
+    - contextPath: CloudShare.POCInvites.status
+      description: Status
+      type: number
+    - contextPath: CloudShare.POCInvites.opportunity
+      description: Opportunity
+      type: string
+    - contextPath: CloudShare.POCInvites.owningSeUserId
+      description: Owning SE user ID
+      type: string
+    - contextPath: CloudShare.POCInvites.projectId
+      description: Project ID
+      type: string
+    - contextPath: CloudShare.POCInvites.randomToken
+      description: Random token
+      type: string
+    - contextPath: CloudShare.POCInvites.id
+      description: ID
+      type: string
+    - contextPath: CloudShare.POCInvites.expirationDate
+      description: Expiration date
+      type: string
+    - contextPath: CloudShare.POCInvites.blueprintName
+      description: Blueprint name
+      type: string
+    description: Retrieves POC invitations sent.
+  dockerimage: cloudshare:latest
+  runonce: false
+  subtype: python3
```

@aburt-demisto If you wish to add the beta label, please commit this suggestion.
```suggestion
  subtype: python3
  beta: true
```